resources/knative-provisioner-natss/charts/knative-provisioner-natss/templates/natss.yaml

```diff
@@ -162,7 +163,9 @@ spec:
         - port: 80
           protocol: TCP
           targetPort: 8080
+          name: http
     selector:
+        app: natss-ch-dispatcher
```

Comment:
Do we really need the app in the selector? The selector should also apply to this pod without relying on the app-label.
If i understand it correctly we put app here just for proper trace naming.

Reply:
No, we don't need selector. Hence removed it as well as the pre-upgrade hook.